src/System.Diagnostics.Process/tests/ProcessTest_ConsoleApp/ProcessTest_ConsoleApp.cs

```diff
@@ -7,45 +8,40 @@ class Program
     {
         static int Main(string[] args)
         {
-            try
+            if (args.Length > 0)
             {
-                if (args.Length > 0)
+                if (args[0].Equals("infinite"))
                 {
-                    if (args[0].Equals("infinite"))
-                    {
-                        // To avoid potential issues with orphaned processes (say, the test exits before
-                        // exiting the process), we'll say "infinite" is actually 30 seconds.
-                        
-                        Console.WriteLine("ProcessTest_ConsoleApp.exe started with an endless loop");
-                        Thread.Sleep(30 * 1000);
-                    }
-                    if (args[0].Equals("error"))
-                    {
-                        Exception ex = new Exception("Intentional Exception thrown");
-                        throw ex;
-                    }
-                    if (args[0].Equals("input"))
-                    {
-                        string str = Console.ReadLine();
-                    }
+                    // To avoid potential issues with orphaned processes (say, the test exits before
+                    // exiting the process), we'll say "infinite" is actually 30 seconds.
+                    Thread.Sleep(30 * 1000);
                 }
-                else
+                else if (args[0].Equals("error"))
+                {
+                    Console.Error.WriteLine("ProcessTest_ConsoleApp.exe error stream");
+                    DelayTask();
+                }
+                else if (args[0].Equals("input"))
+                {
+                    string str = Console.ReadLine();
+                }
+                else if (args[0].Equals("stream"))
                 {
                     Console.WriteLine("ProcessTest_ConsoleApp.exe started");
                     Console.WriteLine("ProcessTest_ConsoleApp.exe closed");
+                    DelayTask();
+                }
+                else
+                {
+                    Console.WriteLine(string.Join(" ", args));
                 }
-
-                return 100;
-            }
-            catch (Exception toLog)
-            {
-                // We're testing STDERR streams, not the JIT debugger. 
-                // This makes the process behave just like a crashing .NET app, but without the WER invocation
-                // nor the blocking dialog that comes with it, or the need to suppress that.
-                Console.Error.WriteLine(string.Format("Unhandled Exception: {0}", toLog.Message));
-                return 1;
             }
+            return 100;
+        }
 
+        public static void DelayTask()
```

Comment:
This "DelayTask" method is just a substitute for Thread.Sleep when Thread.Sleep doesn't exist.  But it does exist here.  I'd prefer to see the other instance of this renamed to ThreadSleep or Sleep or something like that, and then use Thread.Sleep here.